### PR TITLE
Add short link to reference the livestream URL

### PR DESCRIPTION
**Pull request by RealOrangeOne**
## Summary

This means we can just update this URL, rather than needing to update it in a bunch of different locations

### Testing

- [ ] applied the configuration locally
- [ ] manually validated the new behaviour

### CODE REVIEW

File: roles/srobo-nginx/templates/nginx.conf [hunk 1]
```diff
@@ -208,6 +208,9 @@ http {
 
     # Volunteering Links
     rewrite ^/join                 https://docs.google.com/forms/d/e/1FAIpQLSdP55Lz_qNEx7t-hdQctt2VzXdYikOe9k2dKsDzEZZG7-nqTw/viewform redirect;
+    
+    # Short link for livestream embed, mostly for use in kiosk screens and other automation
+    rewrite ^/comp-livestream      https://www.youtube-nocookie.com/embed/7HNG2bqwXlo redirect;
```
Comment (PeterJCLaw) [inline roles/srobo-nginx/templates/nginx.conf:213]: If this is purely an internal thing, suggest we make this year-specific:
```suggestion
    rewrite ^/sr2022-livestream      https://www.youtube-nocookie.com/embed/7HNG2bqwXlo redirect;
```
If this isn't year specific, would a simpler url be easier?
```suggestion
    rewrite ^/livestream      https://www.youtube-nocookie.com/embed/7HNG2bqwXlo redirect;
```

Comment (RealOrangeOne) [reply roles/srobo-nginx/templates/nginx.conf:213]: I like the idea of just having a simple URL for it, we can reuse it between events much easier (and non-comp ones).

**Comment by PeterJCLaw** [comment]
As a general comment -- I've long hoped that we could push this sort of thing into the compstate, though haven't yet worked out how to do that.

**Comment by PeterJCLaw** [comment]
@RealOrangeOne have you tested this at all?

**Comment by PeterJCLaw** [comment]
Closing as this is year-specific and would thus need changes anyway to be useful in the future.

**Pull request closed without merging**